### DISCUSSION TITLE
Pretrained classifier in one of demonstration ipynb files

### DISCUSSION THREAD

**Discussion by 1W1521375** [Trusted-AI/adversarial-robustness-toolbox#1300] [Q&A] [answered]
Hello,
I am new to ART and have a question about a pretrained classifier on CIFAR-10, which is used in the [this](https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/main/notebooks/detection_adversarial_samples_cifar10.ipynb) .ipynb file.

I would like to know whether the classifier loaded in the third block code as `classifier_model` was trained just with the original 50,000 training data from the official CIFAR-10 dataset.

This is the download link in the example: https://www.dropbox.com/s/ta75pl4krya5djj/cifar_resnet.h5?dl=1

Thank you in advance

**Comment by beat-buesser** [comment, marked as answer]
Hi @1W1521375 Yes, I think that model has been trained on the entire CIFAR10 training set (50'000 images).

**Reply by 1W1521375** [reply]
@beat-buesser
I'm glad to hear that... and thank you for the answer.
Is there any other information about this pretrained classifier?
(e.g., readme by its author)

**Reply by beat-buesser** [reply]
@1W1521375 No, unfortunately the mode has been contributed without the training script. We are going to replace it with a better suited model soon in #1298
